### PR TITLE
Trusted build servers moved to AWS and use an elastic IP

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit SvenDowideit@home.org.au

see https://forums.docker.com/t/webhook-ip-changed/2194

### CODE REVIEW

**Comment by thaJeztah** [comment]
Does the note still have value after this change, or should it be removed altogether? (not sure). Thinking of a way to inform users to remove their existing filters, but not sure what's the best way for that.

"lgtm"

ping @fredlf @jamtur01 @moxiegirl wdyt?

**Comment by moxiegirl** [comment]
checking

File: docs/docker-hub/builds.md [hunk 1]
```diff
@@ -398,9 +398,8 @@ Webhooks are available under the Settings menu of each Repository.
 > **Note:** If you want to test your webhook out we recommend using
 > a tool like [requestb.in](http://requestb.in/).
 
-> **Note**: The Docker Hub servers are currently in the IP range
-> `162.242.195.64 - 162.242.195.127`, so you can restrict your webhooks to
-> accept webhook requests from that set of IP addresses.
+> **Note**: The Docker Hub servers use an elastic IP range, so can't be
```
Comment (moxiegirl) [inline docs/docker-hub/builds.md]: Two notes in a row look a bit odd. Rather than form a note list, you can join the top note with the previous sentence. Also, the second note is a bit hard to parse:

-->

Webhooks are available under the "Settings" menu of each Repository. If you want to test your webhook, you can use a tool like [requestb.in](http://requestb.in/).

>**Note**: The Docker Hub servers use an elastic IP range, so you can't filter for them by IP address.

File: docs/docker-hub/repos.md [hunk 2]
```diff
@@ -135,9 +135,8 @@ similar to the example shown below.
 
 For testing, you can try an HTTP request tool like [requestb.in](http://requestb.in/).
 
-> **Note**: The Docker Hub servers are currently in the IP range
-> `162.242.195.64 - 162.242.195.127`, so you can restrict your webhooks to
-> accept webhook requests from that set of IP addresses.
+> **Note**: The Docker Hub servers use an elastic IP range, so can't be
```
Comment (moxiegirl) [inline docs/docker-hub/repos.md]: >**Note**: The Docker Hub servers use an elastic IP range, so you can't filter for them by IP address.

**Comment by thaJeztah** [comment]
@moxiegirl yes, I meant the updated one; the note was previously added to explain which IP-range was used so that users were able to filter the webhooks on IP-address. Now that that is no longer possible, I wondered if there was much added value to mentioning something that *isn't* possible.

Not strongly opposed, so I'm good with merging with the proposed changes.

**Comment by moxiegirl** [comment]
@thaJeztah You have a point I missed that filtering was applied to the webhook.  It is only important to mention the filtering if that is a common task or if Docker Support/user groups are receiving a lot of queries about this feature.  Then, to warn it isn't possible may reduce customer calls/questions and would make sense.  @SvenDowideit probably knows that data better than either of us.

**Comment by moxiegirl** [comment]
LGTM

**Comment by thaJeztah** [comment]
@SvenDowideit I think it's up to you wether or not the note around IP-addresses/filtering should stay in (https://github.com/docker/docker/pull/14574#issuecomment-121021755).

I *do* like the suggestions made by @moxiegirl, especially the one to get rid of two consecutive "notes"; https://github.com/docker/docker/pull/14574#discussion_r34493774

File: docs/docker-hub/builds.md (hunk 1 above)
Comment (SvenDowideit) [inline docs/docker-hub/builds.md]: agreed.

**Comment by SvenDowideit** [comment]
its been a common question, so imo the note has value to answer the readers question when they think it - instead of forcing them to ask :)

the other, I agree, and will update

**Comment by thaJeztah** [comment]
Thanks, and makes sense in that case :-)
